docs/static/persistent-queues.asciidoc

```diff
@@ -55,7 +55,7 @@ The default is 1024mb (1gb).
 +
 Be sure that your disk has sufficient capacity to handle the cumulative total of `queue.max_bytes` across all persistent queues.
 The total of `queue.max.bytes` for _all_ queues should be
-lower than the capacity of your disk. 
+lower than the capacity of your disk. The size is checked at startup.
```

Comment:
IIRC, we check that _each_ queue has sufficient capacity to reach its max size, but we do not validate sufficient room for _all_ to reach max size. It may be worth rewording to indicate this.

Reply:
yes, we do not check space availability for the sum of max_bytes of all queues.